src/Geometry.cpp

```diff
@@ -225,6 +232,17 @@ Geometry::UpdateBuffers() {
   VRB_GL_CHECK(glBindBuffer(GL_ARRAY_BUFFER, 0));
 }
 
+void
+Geometry::SetRenderRange(int32_t aStartIndex, int32_t aLength) {
+  m.rangeStart = aStartIndex;
```

Comment:
Since we know the tri count, it might be good to add some bounds checking here? Ensure start and start + length are in bounds?